server/service/osquery_utils/queries.go

```diff
@@ -699,7 +699,7 @@ var mdmQueries = map[string]DetailQuery{
 		QueryFunc:        buildConfigProfilesWindowsQuery,
 		Platforms:        []string{"windows"},
 		DirectIngestFunc: directIngestWindowsProfiles,
-		Discovery:        discoveryTable("mdm_bridge"),
+		Discovery:        discoveryTableAndServer("mdm_bridge"),
```

Comment:
this rubs me the wrong way, is there any way we can guarantee the `mdm_bridge` to work with the current query instead? why do we need the extra check? is the table crashing before it's registered?

Reply:
The table seems to be loading fine. What seems to be happening is that we're running an MDM command that crashes

Reply:
We run an MDM command that takes the shape of `<Get><CmdID>%s</CmdID><Item><Target><LocURI>%s</LocURI></Target></Item></Get>` every refetch, and that will crash the table and take down orbit and osquery

Reply:
I don't like this workaround either, so I may try to find another solution

Reply:
right on, thanks for the context! what if we fix the `mdm_bridge` table instead to not register itself on windows servers? it's a table we ship and control